gapis/gfxapi/templates/api.go.tmpl

```diff
@@ -127,6 +127,11 @@
     {{$to := Macro "Go.Type" $.To}}
 
     type {{$.Name}} {{$to}}
+    {{if (GetAnnotation $ "dispatchHandle")}}
```

Comment:
Wouldn't it be better to add the dummy method `IsMemoryXXX` if the pseudonym's underlying type is a `int`, `size`, `char`, etc?